docs/data/base/components/checkbox/checkbox.md

```diff
@@ -9,108 +9,78 @@ waiAria: https://www.w3.org/WAI/ARIA/apg/patterns/checkbox/
 
 # Checkbox
 
-<p class="description">Checkboxes give users binary choices when presented with multiple options in a series.</p>
+<p class="description">Checkbox give users a binary choice between multiple options in a series.</p>
```

Comment:
Will leave it in singular form because we need to apply either singular or plural to all components here, and we run into issues with plural. Fixed the typo though.